apps/external-devices/device-descriptors/Yost_3-Space.json

```diff
@@ -0,0 +1,19 @@
+{
+    "deviceVendor": "Yost Labs (YEI)",
```

Comment:
@rpavlik It's an upheaval thing. Yost Labs is a spin-off of sorts that got the tracker hardware while YEI is still its own company.